bin/node/runtime/src/lib.rs

```diff
@@ -615,7 +615,47 @@ impl_runtime_apis! {
 			let mut batches = Vec::<BenchmarkBatch>::new();
 			let params = (&pallet, &benchmark, &lowest_range_values, &highest_range_values, &steps, repeat);
 
+			use pallet_bridge_currency_exchange::benchmarking::{
+				Module as BridgeCurrencyExchangeBench,
+				Trait as BridgeCurrencyExchangeTrait,
+				ProofParams as BridgeCurrencyExchangeProofParams,
+			};
+
+			impl BridgeCurrencyExchangeTrait for Runtime {
+				fn make_proof(
+					proof_params: BridgeCurrencyExchangeProofParams<AccountId>,
+				) -> crate::exchange::EthereumTransactionInclusionProof {
+					use sp_currency_exchange::DepositInto;
+
+					if proof_params.recipient_exists {
+						<Runtime as pallet_bridge_currency_exchange::Trait>::DepositInto::deposit_into(
+							proof_params.recipient.clone(),
+							ExistentialDeposit::get(),
+						).unwrap();
+					}
+
+					let transaction = crate::exchange::prepare_ethereum_transaction(
+						&proof_params.recipient,
+						|tx| {
+							// our runtime only supports transactions where data is exactly 32 bytes long
```

Comment:
In the PR description you elaborate that this is because of Substrate pubkeys, I think it's worth mentioning that in this comment as well